charts/internal/shoot-system-components/charts/cloud-controller-manager/templates/cloud-node-manager-psp-clusterrole.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if semverCompare ">= 1.23" .Capabilities.KubeVersion.GitVersion -}}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: gardener.cloud:psp:kube-system:cloud-node-manager
+rules:
+- apiGroups:
+  - policy
+  - extensions
```

Comment:
is the `extensions` also necessary ?

Reply:
To my understanding K8s 1.15 can still serve the PodSecurityPolicy resource from the `extensions/v1beta1` API version. https://github.com/gardener/gardener-extension-provider-azure#supported-kubernetes-versions still lists 1.15 as technically supported K8s version. We may drop `extensions` as part of the clean up that drops support for the quite old K8s versions (1.15 and 1.16).